### DISCUSSION TITLE
After the child nodes are selected one by one, the parent node is not automatically selected. Is there a solution?

### DISCUSSION THREAD

**Discussion by yxd94772** [dowjones/react-dropdown-tree-select#585] [Q&A]
hello , After the child nodes are selected one by one, the parent node is not automatically selected. Is there a solution?  Is it my problem, or is it designed that way . thanks ~

**Comment by gandhis1** [comment]
Use the option `showPartiallySelected`: https://github.com/dowjones/react-dropdown-tree-select#showpartiallyselected
